app/experimenter/settings.py

```diff
@@ -46,6 +46,7 @@
     "django.contrib.messages",
     "django.contrib.staticfiles",
     "django.forms",
+    "django_filters",
```

Comment:
I'm sort of on the fence about this -- we probably *should* have this here, but we got by without it, and now the only thing it provides is a template that we override, so do we really need it?

Reply:
Made this change!